locales/de.json

```diff
@@ -119,6 +119,7 @@
 		"button": {
 			"changePassword": "Passwort ändern",
 			"consent": "Einwilligungen:",
+			"consentNotNecessary": "Registrierung:",
```

Comment:
I think the naming does not reflect the wording very good even though I agree on making the distinction clear that this has to do with the not needed consent. Maybe instead of making it named to the consent, make it linking to the actual action needed, so in this case: "onlyRegistrationNeeded". At least thats my two cents ^^